src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -82,4 +82,16 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_DotNet)' != 'true'
```

Comment:
I think we should move the Visual Basic `Imports` from Microsoft.NET.Sdk.VisualBasic.targets to this file, and deduplicate the ones that are shared.

Reply:
I was planning on doing that separately to minimize the risk of delaying this PR would that be okay?

Reply:
Yes, that would be fine